FirstAnalysis/HFmcvalidation.py

```diff
@@ -111,10 +157,13 @@ def decayLength_reco_MC(file):
     hDecLenRes = file.Get("hf-mc-validation-rec/histDecLen")
     hDecLenRes.Rebin(2)
     hDecLenRes.Draw()
-    printCanvas(cDecLenRes, "Resolution_DecayLength")
+    printCanvas(cDecLenRes, "BtoDForcedProduction/Resolution_DecayLength")
 
 
-fileo2 = TFile("../codeHF/CCbar_pp_AnalysisResults_O2_500files.root")
+# fileo2 = TFile("/data/Run5/plots/new_pro/AnalysisResults_O2_pp14TeV_ccbar_10M_sc3werner_20042021.root")
+fileo2 = TFile(
+    "/home/mmazzill/alice/Run3Analysisvalidation/codeHF/AnalysisResults_O2_LcpKpi_4_6.root"
```

Comment:
Can you remove local paths? For the moment is fine if it refers to ../codeHF/xxx